src/pages/extending-agent-app/products-sdk/helpdesk-sdk/index.mdx

```diff
@@ -0,0 +1,504 @@
+
+It displays a notification badge on top of your app’s icon:
+
+<p style={{maxWidth: "70px"}}>
+  <img src="/images/extending-agent-app/fullscreen-widget-badge.png" alt="Fullscreen widget badge" width="70px" height="190px" />
+</p>
+
+If you want to remove the badge, invoke the method with the `null` parameter.
+
+<CodeSample>
+
+```js
+widget.setNotificationBadge(null);
+```
+
+</CodeSample>
+
+Make sure Agents can dismiss the notification to avoid cluttered UI.
+
+# Advanced use
+
+## Developing your own widget
+
+If you're building your own widget and you need to interact with the HelpDesk App, be sure to use the [HelpDesk SDK](#top).
+
+## Accessing HelpDesk data
+
+You can leverage the OAuth 2.1 authorization flow to use <a href="https://api.helpdesk.com/docs" target="_blank">HelpDesk API</a>. Head to [Sign in with LiveChat](/getting-started/authorization/sign-in-with-livechat/) docs for more information.
+
+## Layout and Styling
+
+If you're using React, you can use the [LiveChat Design System](/extending-ui/extending-agent-app/design-system/), which is a component library used directly in the LiveChat user interface.
+
+## Hosting the widget
+
+You can host your widget locally or on a dedicated server. The hosted content has to be served over the **HTTPS Protocol**.
+
+During development, you can use a self-signed certificate for `localhost` or upload your widget to an SSL-enabled host. You can also leverage bundlers like [Webpack](https://webpack.js.org/configuration/dev-server/) to use an https-enabled development server.
+
+If you need a quick way to **deploy** your widget, you can use [Netlify Drop](https://developers.livechat.com/blog/widget-hosting-on-netlify/).
+
+## Creating a Ticket Details widget
+
+When developing an app in the **Details section**, you can add some content to the **Opening state** and display it in the **Opening state preview**. This way, you turn it into a Ticket Details widget.
+
+Let's define the terms we use in this document:
+
+- **Section** - an element of Ticket Details. It contains a complete set of information grouped under a single widget.
+- **Component** - a single line in a section. It can have one of the pre-defined formats and be filled with data from the app.
+
+### How to add a widget
+
+To display a widget, we have introduced the **Opening state**, which allows you to add components to your application and see a live preview of what the app will look like in the **Opening state preview**.
+
+The components you may add are:
+
+- [Header](/extending-agent-app/products-sdk/helpdesk-sdk#header)
+- [Item](/extending-agent-app/products-sdk/helpdesk-sdk#item)
+- [Button](/extending-agent-app/products-sdk/helpdesk-sdk#button)
+- [Label with value](/extending-agent-app/products-sdk/helpdesk-sdk#label-with-value)
+- [Link](/extending-agent-app/products-sdk/helpdesk-sdk#link)
+- [Divider](/extending-agent-app/products-sdk/helpdesk-sdk#divider)
+
+<div style={{maxWidth: '800px', marginBottom: '30px'}}>
```

Comment:
I wonder if we should duplicate the same image (it's already used in the Building HelpDesk apps document if I remember correctly).  One thing about this image is that it's really big 🤔 I don't know

Reply:
It is added in the App Guides, in [HelpDesk Widgets BB](https://ded-337--livechat-public-docs.netlify.app/extending-agent-app/products-sdk/helpdesk-sdk#creating-a-ticket-details-widget). 
I put it more like a test as I was not sure about it, and since it disturbed you while reading then, my second option is that maybe we can link here that HelpDesk Widget BB? Instead of a pic, we can write - `To learn more about the HelpDesk Widgets, please read here`? WDYT? 🤔 
I believe that this visual might be useful, and the link might be a nice solution here.

Reply:
@ale-kacperczyk I have changed the image for a link, please see if that looks better (For me it does) :)